lib/code_corps/stripe_service/webhook_processing/ignored_event_handler.ex

```diff
@@ -32,21 +47,36 @@ defmodule CodeCorps.StripeService.WebhookProcessing.IgnoredEventHandler do
+    with ignored_reason <- get_reason(type, handler) do
       local_event |> set_ignored(ignored_reason)
     end
   end
 
-  @spec get_reason(String.t) :: String.t
-  defp get_reason("account.external_account.created"), do: "External accounts are stored locally upon updating a connect account."
-  defp get_reason("application_fee.created"), do: "We don't make use of the application fee object."
-  defp get_reason("customer.created"), do: "Customers are only created from the client."
-  defp get_reason("customer.source.created"), do: "Cards are only created from the client. No need to handle"
-  defp get_reason("customer.subscription.created"), do: "Subscriptions are only created from the client."
-  defp get_reason("invoice.created"), do: "We prefer to handle other lifecycle events for invoices, like payment_succeeded."
-  defp get_reason("plan.created"), do: "Plans are only created from the client."
+  @spec get_reason(String.t, Module.t) :: String.t
+  defp get_reason(type, ConnectEventHandler), do: get_connect_reason(type)
+  defp get_reason(type, PlatformEventHandler), do: get_platform_reason(type)
+
+  @spec get_connect_reason(String.t) :: String.t
+  defp get_connect_reason("account.external_account.created"), do: "External accounts are stored locally upon updating a connect account."
+  defp get_connect_reason("application_fee.created"), do: "We don't make use of the application fee object."
+  defp get_connect_reason("customer.created"), do: "Customers are only created from the client."
+  defp get_connect_reason("customer.updated"), do: "We already propagate connect customer updates when a platform customer update is handled."
+  defp get_connect_reason("customer.source.created"), do: "Cards are only created from the client. No need to handle"
+  defp get_connect_reason("customer.subscription.created"), do: "Subscriptions are only created from the client."
+  defp get_connect_reason("invoice.created"), do: "We prefer to handle other lifecycle events for invoices, like payment_succeeded."
+  defp get_connect_reason("plan.created"), do: "Plans are only created from the client."
+
+  @spec get_platform_reason(String.t) :: String.t
+  defp get_platform_reason("account.external_account.created"), do: "External accounts are stored locally upon updating a connect account."
+  defp get_platform_reason("application_fee.created"), do: "We don't make use of the application fee object."
+  defp get_platform_reason("customer.created"), do: "Customers are only created from the client."
+  defp get_platform_reason("customer.source.created"), do: "Cards are only created from the client. No need to handle"
+  defp get_platform_reason("customer.subscription.created"), do: "Subscriptions are only created from the client."
+  defp get_platform_reason("invoice.created"), do: "We prefer to handle other lifecycle events for invoices, like payment_succeeded."
+  defp get_platform_reason("plan.created"), do: "Plans are only created from the client."
```

Comment:
I'm aware there is some code duplication here and we could reduce it but I feel it's clearer to keep the mapping of messages to event types separate. It also allows us to change the actual text based on the type of webhook that got hit.